solidus_volume_pricing.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |s|
   s.require_path = 'lib'
   s.requirements << 'none'
 
-  s.add_runtime_dependency 'solidus_core', '~> 1.3'
+  s.add_runtime_dependency 'solidus_core', '~> 2.1'
```

Comment:
Yes, that makes sense. Additionally, changing the gemfile back to
branch = ENV.fetch('SOLIDUS_BRANCH', 'v1.3') 
fixes the strange Active Support error. I'll make the changes.